src/parser/DASHTree.cpp

```diff
@@ -132,6 +134,8 @@ static unsigned int ParseSegmentTemplate(const char** attr,
   {
     if (!tpl.initialization.empty() && tpl.initialization[0] == '/')
```

Comment:
As <!tpl.initialization.empty() > was checked in the parent IF block, the check here wouldn't be required.